services/web-app/components/mdx-notification/mdx-notification.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { ActionableNotification, Button, Column, Grid, Link } from '@carbon/react'
+import { ChevronDown, ChevronUp } from '@carbon/react/icons'
+import clsx from 'clsx'
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+
+import styles from './mdx-notification.module.scss'
+
+export const MdxNotification = ({
+  content,
+  title,
+  description,
+  href,
+  link,
+  id,
+  kind = 'error',
+  collapsible = false
+}) => {
+  const wrapperClassName = clsx({
+    'cds--inline-notification': true
+  })
+
+  const [collapsed, setIsCollapsed] = useState(false)
+
+  return (
+    <Grid narrow className={styles['error-container']}>
+      <Column sm={4} md={6} lg={16}>
```

Comment:
```suggestion
      <Column sm={4} md={8} lg={8}>
```
Looks like the Figma file shows it at 8 col? and I believe it should match the col width of the text at md. 


![Screen Shot 2022-10-07 at 7 14 37 AM](https://user-images.githubusercontent.com/2753488/194550959-cf03ff0b-7586-4021-84bf-e78d755ff137.png)